### ISSUE TITLE
Can't use Min Time with trial type Likert

### ISSUE THREAD

**Issue by lukechurchill** [gikeymarcia/Collector#163]
Hi!

I'm trying to impose a minimum time of 1 second on a likert trial, but it doesn't seem to be working. I've tested other trial types, including Instruct, JOL, and multipleChoiceText, and min time works for all of them. What should I do?

Best,

Luke

**Comment by TysonKerr** [comment]
Hmm, sounds like a bug. I'll fix it. Are you using the master branch of collector?

**Comment by TysonKerr** [comment]
I found a bug in master. It should be fixed now, so if you redownload the collector, min time should work on likert trials. More specifically, you can just look inside the Code folder, inside the TrialTypes folder, and then inside the Likert folder for the display.php file. In there, I realized that the submit button didn't have the id "FormSubmitButton", which the code for min timing was looking for. So, by adding the correct ID to the submit button in the Likert trial, it can use min timing now.

basically, `Code/TrialTypes/Likert/display.php` had the change on line `124`

``` html
<button type="submit" class="collectorButton collectorAdvance">Submit</button>
```

to 

``` html
<button type="submit" id="FormSubmitButton" class="collectorButton collectorAdvance">Submit</button>
```

**Comment by lukechurchill** [comment]
Thank you!
